static/less/shared-components.less

```diff
@@ -376,7 +376,7 @@ table.table.key-value {
 .org-avatar {
   width: 30px;
   height: 30px;
-  background: @red url(../images/logos/sentry-avatar.png);
+  background: @red url(../images/sentry-avatar.png);
```

Comment:
Not sure why this was in `logos/` in the first place, but moved to static/images